hack/verify-govet.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${KUBE_ROOT}/hack/lib/init.sh"
+
+cd "${KUBE_ROOT}"
+
+# Use eval to preserve embedded quoted strings.
+eval "goflags=(${KUBE_GOFLAGS:-})"
+
+# Filter out arguments that start with "-" and move them to goflags.
+targets=()
+for arg; do
+  if [[ "${arg}" == -* ]]; then
+    goflags+=("${arg}")
+  else
+    targets+=("${arg}")
+  fi
+done
+
+if [[ ${#targets[@]} -eq 0 ]]; then
+  targets=`go list -f '{{.ImportPath}}' ./...`
```

Comment:
You could also filter out `third_party` here instead of having to do a for loop below.